WalletWasabi/BitcoinP2p/TrustedP2pBehavior.cs

```diff
@@ -37,7 +37,7 @@ protected override bool ProcessInventoryVector(InventoryVector inv, EndPoint rem
 			return true;
 		}
 
-		if (inv.Type.HasFlag(InventoryType.MSG_WITNESS_BLOCK))
+		if (inv.Type.HasFlag(InventoryType.MSG_BLOCK))
```

Comment:
Inventory messages only come with TX or BLOCK, never MSG_WITNESS_BLOCK.

Reply:
What about WTX? If they also don't come with WTX then you forgot to change the thing some lines above, as well as in `UntrustedP2pBehavior`: `if (inv.Type.HasFlag(InventoryType.MSG_TX) || inv.Type.HasFlag(InventoryType.MSG_WTX))`